src/main/java/Duke.java

```diff
@@ -1,10 +1,139 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    // Constants that store index no. of "deadline" and "event"
+    public static final int DASH_INDX     = 4;
+    public static final int TODO_SIZE     = 5;
+    public static final int EVENT_SIZE    = 6;
+    public static final int DEADLINE_SIZE = 9;
+    public static final Task[] tasks      = new Task[100];
```

Comment:
Would it be better to use `100` as a constant?
```suggestion
public static final int TASK_ARRAY_SIZE = 100;
public static Task[] tasks = new Task[TASK_ARRAY_SIZE];
```